eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/SegmentNr.scala

```diff
@@ -71,6 +160,17 @@ object SegmentNr {
       .toList
```

Comment:
why deprecated implementation doesn't call the new one?

Reply:
nice catch, thanks, fixed